### PR TITLE
Big renaming

### PR DESCRIPTION
**Pull request by MrBrain295**
Closes: #38 
Some I couldn't find the name and I am not going to rename sum.b.

### CODE REVIEW

**Comment by clarkzjw** [comment]
@MrBrain295 Hi, sum.b is an implementation in [brainfuck](https://en.wikipedia.org/wiki/Brainfuck)

**Comment by MrBrain295** [comment]
sorry I am not going to do that one

**Comment by clarkzjw** [comment]
Ok, that's fine. 

And actually the name of languages can be found in README.md and could you please update the links in README.md or modify it in a way as well? Thanks.

**Comment by MrBrain295** [comment]
ok

**Comment by MrBrain295** [comment]
@clarkzjw done

File: README.md [hunk 1]
```diff
@@ -3,37 +3,37 @@
...
-- [Rust](sum.rs)
-- [Scala](sum.scala)
-- [Shell](sum.sh)
-- [SML](sum.sml)
-- [Smalltalk](sum.st)
-- [Swift](sum.swift)
-- [Futhark](sum.fut)
+- [C](C.c)
+- [C++](C++.cpp)
+- [C#](C Sharp.cs)
```
Comment (clarkzjw) [inline README.md:14]: Escape space like Common Lisp below.

**Review by clarkzjw — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MrBrain295) [reply README.md]: @clarkzjw done

**Review by clarkzjw — APPROVED** [review]

**Comment by clarkzjw** [comment]
@MrBrain295 Thanks!

**Pull request merged**